CIPs/CIP-79/cip-79.md

```diff
@@ -0,0 +1,270 @@
+---
+cip: 79
+title: 3ID DID Method Specification
+author: Joel Thorstensson (@oed)
+discussions-to: https://github.com/ceramicnetwork/CIP/issues/80
+status: Draft
+category: Standards
+type: RFC
+created: 2021-02-12
+---
+
+# 3ID DID Method Specification
+
+3ID is a DID method that is implemented natively on Ceramic. It uses a Ceramic document as a source of truth of the information which makes up the DID document for the 3ID. Documents in Ceramic achieves secure key rotation though *proof-of-publication* by anchoring document updates into a blockchain. This means that 3ID inherits this property.
+
+## DID Method Name
+
+The name string that shall identify this DID method is: `3`.
+
+A DID that uses this method MUST begin with the following prefix: `did:3`. Per the [DID specification](https://w3c.github.io/did-core/), this string MUST be in lowercase. The remainder of the DID, after the prefix, is specified below.
+
+## Method Specific Identifier
+
+There are two versions of 3IDs, both versions use a Ceramic document as a way to update the DID document. 3IDv1 is the most recent version, 3IDv0 is supported for legacy reasons. If you are creating new 3IDs you should always use 3IDv1.
```

Comment:
Do we need to explain how to know if a 3ID is v0 or v1?  I assume we try to parse the identifier as a DocID and if that fails then we parse it as a CID and assume we have a v0 id?

Reply:
Good point. I'll add an explainer for this 👍